README.md

```diff
@@ -1,19 +1,39 @@
 ![activitypods-small](https://user-images.githubusercontent.com/17931931/215525902-6ae72fa9-fde0-43eb-a053-0ccfd4565ead.png)
 
-# ActivityPods shapes repository
+# ActivityPods shapes Monorepo
 
-A repository with shapes and shape trees used by [ActivityPods](https://activitypods.org) applications.
+A monorepo with RDF shapes, shape trees, and LDO objects used by [ActivityPods](https://activitypods.org) applications.
 
-## Running locally
+## Serving shapes
+
+When developing a new application, you can serve your new shapes locally. When you are serving your app (built from the template) and pod provider in development mode, the client will notice that a local shape server is running on this port (`30916`) and use this server as the shape provider instead of https://shapes.activitypods.org/.
 
 ```bash
 yarn install
 yarn start
 ```
 
+## Packages
+
+This monorepo has the following structure:
+
+- `packages/shape-definitions`: Contains the shape definitions and exports to js/ts.
+- `packages/shape-definitions/source/<version/`: Contains the actual (SHACL) shape and shape-tree definitions serialized in turtle (`.shacl.ttl`). Also contains SHEX shape files: For the moment, they are generated manually and only contain ActivityStreams translations that are derived from the SHACL definitions. We need the SHEX files to generate ldo objects from them.
+- `packages/shape-definitions/generated/` contains js-importable json-ld and ttl-string exports generated from the source `.shacl.ttl` files.
```

Comment:
Do we need these "generated" files for LDO or is it for other usages ?

Reply:
Normally, you can't import non-js/json files from a package. So this is why I exported them this way.
We don't necessarily need it in jsonld though.